### ISSUE TITLE
<start.followInNewSplit>

### ISSUE THREAD

**Issue by se7en-x230** [Jelmerro/Vieb#321]
Can I map a key similar as f <start.FollowNewTab> that does something like <start.followInNewSplit> to open link in split and or be added to the right click menu?

**Comment by se7en-x230** [comment]
This refers to #198 not working anymore.
`pmap * <pointer.copyLink><:vsplit><action.openFromClipboard>`

Copies the url in my clipboard, splits the screen, opens the link in the original window and goes into follow mode.

So I guess something changed in the overall setup as it use to work just fine with this entry.

Opening a link side by side is something valuable if you compare sites/products and so on.

**Comment by se7en-x230** [comment]
`pmap 9 <pointer.copyLink><action.toNormalMode><:vsplit><action.toExploreMode><action.openFromClipboard>
`
That works for whatever reason ...

**Comment by Jelmerro** [comment]
I have added two new follow mode types, one for opening regular splits and one for opening vertical splits (vsplits). There are also many new link-related pointer mode actions as well as context menu options in various places to open links in new splits or vsplits. Let me know if you still need anything to remove your workaround or would like additional default mappings, but for convenience I have already added `*` as a default mapping to open a new vertical split from the link at the pointer location.
